kubernetes/helm_charts/core/print/values.j2

```diff
@@ -0,0 +1,49 @@
+#jinja2:lstrip_blocks: True
+
+### Default variable file for print-service ###
+
+namespace: {{ namespace }}
+imagepullsecrets: {{ imagepullsecrets }}
+dockerhub: {{ dockerhub }}
+
+replicaCount: {{print_replicacount|default(1)}}
+repository: {{print_repository|default('print-service')}}
+image_tag: {{ image_tag }}
+resources:
+  requests:
+    cpu: {{print_cpu_req|default('50m')}}
+    memory: {{print_mem_req|default('50Mi')}}
+  limits:
+    cpu: {{print_cpu_limit|default('1')}}
+    memory: {{print_mem_limit|default('1024Mi')}}
```

Comment:
Set default to 500 Mi. Any reason to set min of 1Gb?

Reply:
it's running chrome inside. So it'll be good to have 1G limit